ngraph/core/reference/include/ngraph/runtime/reference/detection_output.hpp

```diff
@@ -397,7 +402,7 @@ namespace ngraph
                         }
                     }
 
-                    std::stable_sort(
+                    std::sort(
```

Comment:
How this change helps to resolve ambiguity? I think that the stable_sort is more appropriate rather than regular sort.

Reply:
It doesn't help per se - with new comparator none of the elements will ever be considered equal by the sorting algorithm, so having the stable version is redundant.

Reply:
@tkrupa-intel What if there are two elements that have matching pairs?

Reply:
This will never happen (at least in the current state) because all in of the calls using this comparator, the second element of the pair contains the ordinal index.

Reply:
@tkrupa-intel Still, I think we need to be future-proof. I can easily imagine e.g. some DetectionOutputLite where for the sake of minimality there are no indices.

Reply:
Not using indices would be ideal, but in that case we're back in square one - without setting a comparator rule with indices (or guaranteeing sort stability in some other way) there is potentially more than one correct set of results, which is unacceptable if one wants to utilize functional tests. Is there some other solution I'm not seeing?

Reply:
I`m afraid there is no universal solution here, but please do retain `std::stable_sort`.

Reply:
Resolved.